src/openrct2/network/NetworkTypes.h

```diff
@@ -126,7 +126,7 @@ template<typename T, size_t _TypeID> struct NetworkObjectId_t
 };
 #pragma pack(pop)
 
-using NetworkRideId_t = ride_id_t;
+using NetworkRideId_t = RideId;
```

Comment:
Why is we need a separate NetworkRideId_t type?

Reply:
We don't, it used to be a different thing prior to ride_id_t becoming enum.